app/views/issues/_table.html.erb

```diff
@@ -16,7 +16,7 @@
       </thead>
       <tbody>
         <% issues.each do |issue| %>
-        <% cache [issue, @columns, issue.affected_count, 'issues-table'] do%>
+        <% cache [issue, @columns, issue.affected_count, issue.state, 'issues-table'] do %>
```

Comment:
wouldn't a change in the `issue.state` be part of a change in the `issue` (first parameter of the key)

Reply:
Not needed anymore. See comment: https://github.com/dradis/dradis-ce/pull/659#discussion_r422035116